spec/factories/profiles.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :profile do
+    job { "Devops egineers" }
```

Comment:
user { association :user } should be added?

Reply:
done